docs/user-guide/creating-application/workflow/ci-pipeline.md

```diff
@@ -2,9 +2,8 @@
 
 ## 1. Continuous Integration
 
-CI related configuration is provided under this section. We will later discuss this in detail.
-
-Click on `Continuous Integration` to configure CI Pipeline, a new window will appear. You can see 5 options on this new window:
+CI related configuration is provided under this section. We will later discuss this in detail. 
```

Comment:
CI related configuration is provided under this section. Keys concepts of CI pieline are: